.github/ISSUE_TEMPLATE/safe-app-request.md

```diff
@@ -4,17 +4,28 @@ about: Create a request to add, remove or update a Safe app
 ---
 
 <!--
-BEFORE SUBMITTING: Please search to make sure this request has not been opened already
+BEFORE SUBMITTING:
+    1. Please search to make sure this request has not been opened already
+    2. Please make sure that you followed the instructions at https://docs.gnosis-safe.io/build/sdks/safe-apps/releasing-your-safe-app
 -->
 
+### Revision checks
+<!--
+Please tell us if you did at least this checks
+-->
+ - [ ] Your Safe App include a manifest.json file at the root [with the required data](https://docs.gnosis-safe.io/build/sdks/safe-apps/releasing-your-safe-app)
```

Comment:
WDYT about adding "can be loaded as a custom app"? that would mean CORS is correctly configured